scripts/likelyLocalesAdditional.json

```diff
@@ -1,4 +1,33 @@
 {
+    "142": "zh-Hans-CN",
```

Comment:
we don't support number-style locales. Should we addthese locales as well?

Reply:
They were there before in v41, so I figured we should continue to support them.